### ISSUE TITLE
[Bug]: [Nightly]Milvus crashed with panic 'failed to update next target for collection'

### ISSUE THREAD

**Issue by NicoYuan1986** [milvus-io/milvus#20517]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: cd19d99
- Deployment mode(standalone or cluster): standalone
- SDK version(e.g. pymilvus v2.0.0rc2): 2.2.0.dev72
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Milvus crashed with message 'failed to update next target for collection'.
```
2022-11-11T02:49:54.253204066+08:00 stdout F [2022/11/10 18:49:54.253 +00:00] [ERROR] [meta/coordinator_broker.go:134] ["get recovery info failed"] [collectionID=437283305958574773] [partitionID=437283305958574779] [error="code: CollectionNotExists, reason: can't find collection: 437283305958574773"] [stack="github.com/milvus-io/milvus/internal/querycoordv2/meta.(*CoordinatorBroker).GetRecoveryInfo\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/coordinator_broker.go:134\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).PullNextTarget\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:152\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).updateCollectionNextTarget\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:125\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).UpdateCollectionNextTarget\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:118\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recoverCollectionTargets\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:516\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recover.func1\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:501\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57"]
2022-11-11T02:49:54.253223875+08:00 stdout F [2022/11/10 18:49:54.253 +00:00] [ERROR] [meta/target_manager.go:127] ["failed to get next targets for collection"] [collectionID=437283305958574773] [error="code: CollectionNotExists, reason: can't find collection: 437283305958574773"] [stack="github.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).updateCollectionNextTarget\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:127\ngithub.com/milvus-io/milvus/internal/querycoordv2/meta.(*TargetManager).UpdateCollectionNextTarget\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/meta/target_manager.go:118\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recoverCollectionTargets\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:516\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recover.func1\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:501\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57"]
2022-11-11T02:49:54.253243782+08:00 stdout F [2022/11/10 18:49:54.253 +00:00] [ERROR] [querycoordv2/server.go:519] ["failed to update next target for collection"] [error="code: CollectionNotExists, reason: can't find collection: 437283305958574773"] [stack="github.com/milvus-io/milvus/internal/querycoordv2.(*Server).recoverCollectionTargets\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:519\ngithub.com/milvus-io/milvus/internal/querycoordv2.(*Server).recover.func1\n\t/go/src/github.com/milvus-io/milvus/internal/querycoordv2/server.go:501\ngolang.org/x/sync/errgroup.(*Group).Go.func1\n\t/go/pkg/mod/golang.org/x/sync@v0.0.0-20210220032951-036812b2e83c/errgroup/errgroup.go:57"]
2022-11-11T02:49:54.255373473+08:00 stderr F panic: failed to update next target for collection[ShowPartitions failed: code: CollectionNotExists, reason: can't find collection: 437283305958574074]
2022-11-11T02:49:54.255383943+08:00 stderr F
2022-11-11T02:49:54.25538662+08:00 stderr F goroutine 327 [running]:
2022-11-11T02:49:54.255389401+08:00 stderr F github.com/milvus-io/milvus/cmd/components.(*QueryCoord).Run(0xc00139b680?)
2022-11-11T02:49:54.255391969+08:00 stderr F    /go/src/github.com/milvus-io/milvus/cmd/components/query_coord.go:54 +0x56
2022-11-11T02:49:54.255394547+08:00 stderr F github.com/milvus-io/milvus/cmd/roles.runComponent[...].func1()
2022-11-11T02:49:54.255404153+08:00 stderr F    /go/src/github.com/milvus-io/milvus/cmd/roles/roles.go:97 +0x15c
2022-11-11T02:49:54.255406161+08:00 stderr F created by github.com/milvus-io/milvus/cmd/roles.runComponent[...]

```

### Expected Behavior

Pass

### Steps To Reproduce

```markdown
https://jenkins.milvus.io:18080/blue/organizations/jenkins/Milvus%20Nightly%20CI/detail/master/189/pipeline/123
```

### Milvus Log

[artifacts-milvus-standalone-nightly-189-pymilvus-e2e-logs.tar.gz](https://jenkins.milvus.io:18080/job/Milvus%20Nightly%20CI/job/master/189/artifact/artifacts-milvus-standalone-nightly-189-pymilvus-e2e-logs.tar.gz)

**Comment by yanliang567** [comment]
@NicoYuan1986 is this one master branch or 2.2 branch?

**Comment by NicoYuan1986** [comment]
> @NicoYuan1986 is this one master branch or 2.2 branch?

master

**Comment by weiliu1031** [comment]
/assign @NicoYuan1986 
please verify on this

**Comment by NicoYuan1986** [comment]
Fixed！

**Comment by cangyin** [comment]
@NicoYuan1986 How do you fixed that?

**Comment by NicoYuan1986** [comment]
> @NicoYuan1986 How do you fixed that?

The issue has been fixed by pr #20553.  Have you encountered a similar issue to this?

**Comment by cangyin** [comment]
Yes! I'll try upgrading. Thanks.
